gazebo_msgs/ModelState robotPose
---
